procrastinate/aiopg_connector.py

```diff
@@ -87,8 +87,11 @@ def __init__(
     ):
         """
         Create a PostgreSQL connector using aiopg. The connector uses an ``aiopg.Pool``,
-        which is either created automatically upon first use, or set through the
-        `AiopgConnector.set_pool` method.
+        which is either created by `App.open_async`.
```

Comment:
Remove "either" if there is no option.... or something is missing

Reply:
I wouldn't mention `App.open_sync` in the `AiopgConnector` docstrings. I'd just say that the pool is created internally, or set into the connector, by calling `open_async`.

Reply:
Updated